cdap-hbase-compat-base/src/main/java/co/cask/cdap/data2/util/hbase/DefaultHBaseDDLExecutor.java

```diff
@@ -45,9 +45,9 @@
 public abstract class DefaultHBaseDDLExecutor implements HBaseDDLExecutor {
   public static final Logger LOG = LoggerFactory.getLogger(DefaultHBaseDDLExecutor.class);
   public static final long MAX_CREATE_TABLE_WAIT = 5000L;    // Maximum wait of 5 seconds for table creation.
-  private final Configuration hConf;
+  private Configuration hConf;
 
-  public DefaultHBaseDDLExecutor(Configuration hConf) {
+  void setHConf(Configuration hConf) {
```

Comment:
Adding `initialize(Configuration hConf)` will add dependency on the hadoop though. Should we have `Configuration` class in the spi as well? or should this method just take `Map<String, String> properties`, where properties will include properties from both `Configuration` and `CConfiguration`?